pkg/kudoctl/cmd/plan/plan_history.go

```diff
@@ -79,14 +62,14 @@ func planHistory(args []string, options *historyOptions) error {
 
 	var labelSelector string
 	if len(args) == 0 {
-		fmt.Printf("History of all plan-executions for instance \"%s\" in namespace \"%s\":\n", options.instance, options.namespace)
-		labelSelector = "instance=" + options.instance
+		fmt.Printf("History of all plan-executions for instance \"%s\" in namespace \"%s\":\n", options.Instance, options.Namespace)
+		labelSelector = "instance=" + options.Instance
```

Comment:
for clarity, use fmt.Sprintf instead of joining strings with +. The whole format becomes a lot more clear, especially in for line 69.

Reply:
this was previous code that was refactored to be consistent etc.  I didn't comb through it in detail... glad you did... makes sense!